compose/install.md

```diff
@@ -51,7 +51,7 @@ Python package manager or installing Compose as a container.
 
 ### Install Compose on macOS
 
-**Docker Desktop for Mac** and **Docker Toolbox** already include Compose along
+**Docker Desktop for Mac** and **Docker Toolbox** already includes Compose along
```

Comment:
I don't think it's correct, because it's "Docker Desktop for Mac and Docker Toolbox" (plural), in which case "include" is correct